### DISCUSSION TITLE
How do I import a state file?

### DISCUSSION THREAD

**Discussion by markboyall** [reduxjs/redux-devtools#1062] [Q&A]
We embedded functionality in our application for recording state files, which we imported into the Redux dev tools later for debugging. I haven't had to use it for a while. Clearly the devtools have received some updates since then. Now I cannot find how I'm supposed to import the state file. There doesn't seem to be an import button and I can't drag and drop either. How do I import the state?

**Comment by Methuselah96** [comment]
It should be the second button from the bottom-left:
![image](https://user-images.githubusercontent.com/693755/151913166-86976c7f-343f-4809-81bd-05fd94044f46.png)

**Reply by markboyall** [reply]
@Methuselah96 I realise I posted my other comment as an answer; please see below which is that the import buttons simply don't appear. Nor do I get the "no store detected" message. We definitely create a store and this same code worked on previous versions of the dev tools.

**Comment by markboyall** [comment]
@Methuselah96 I don't have any such button. There is nothing between RTK Query, and the windowing options. I checked the features but this object is not set, so it should default to true.
![image](https://user-images.githubusercontent.com/17589164/154077237-393f13d0-6b7b-43ff-8af0-9cf6eb042a80.png)
